src/screens/ParadeMapScreen/Map.js

```diff
@@ -23,7 +23,7 @@ import locationButtonActive from "../../../assets/images/location-active.png";
 
 type PermissionStatus = "authorized" | "denied" | "restricted" | "undetermined";
 const neverAskStatuses = Platform.select({
-  android: new Set(["restricted"]),
+  android: new Set(),
```

Comment:
I do see where you're coming from, but I don't think it does work quite as expected with restricted in the never ask statuses for android. I tested again on my phone this morning, deleted the app, made sure to reset my location permissions, and was returned restricted on first opening of the app. However, when removing restricted from the Set, the never ask again option does still work, i.e. the location button is showing but doesn't work (which of course is not ideal).
I will leave restricted in the set for now and see how Andrei gets on with testing, but in the event that the behaviour I'm getting persists, do you have any suggestions as to how best to tackle the issue?

Reply:
Interesting. This is definitely worth testing. When uninstall the app, it resets the permissions for it. After installing the app again, the permissions return "undetermined".

Reply:
I wonder if that's a bug in the permission library. I can see that it uses local storage to save if it asked for permissions before ([here](https://github.com/yonahforst/react-native-permissions/blob/master/lib/permissions.android.js#L35)). If this is not cleared correctly, it would return "restricted" immediately.

Reply:
Yeah, I think that's the issue: https://github.com/yonahforst/react-native-permissions/issues/128